### PR TITLE
Added Home page

### PR DESCRIPTION
**Pull request by Nagashree01**
Created  Homepage
homepage-ecom

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+*{
```
Comment (sameep-s) [inline styles.css:1]: you can add these styles in a utility class in your component library and use it in you every project for uniformity.

**Review by sameep-s — APPROVED** [review]
Everything looks good.

File: styles.css (hunk 1 above)
Comment (Nagashree01) [reply styles.css:1]: Yes will do this

File: index.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    align-items: center;
+    justify-content: center;
+    width: 20%;
+}
+.explore-btn:hover{
+    background-color: var(--primary-color);
+    color: var(--white-color);
+}
+.card-horizontal{
+    width: 25%;
```
Comment (JantuDeb) [inline index.css:33]: You can use  `max-width` so it will not shrink on smaller screen.

**Review by JantuDeb — APPROVED** [review]
Everything is looking good, added a suggestion though.

File: index.css (hunk 2 above)
Comment (Nagashree01) [reply index.css:33]: Yes will do this

**Pull request merged**